### ISSUE TITLE
Orgue déménagé à Saint-Martin-le-Vieil

### ISSUE THREAD

**Issue by CelestinHuet** [inventaire-des-orgues/indexes#305]
L'orgue des soeurs bénédictines d'Angers a été déménagé à Saint-Martin-le-Vieil (Aude)
https://inventaire-des-orgues.fr/detail/orgue-angers-soeurs-benedictines-du-calvaire-chapelle-fr-49007-anger-soeurs2-x/

**Comment by CelestinHuet** [comment]
Cas où l'orgue provient de l'étranger : mettre la commune de la provenance en facultatif ?

**Comment by GwilhermPoullennec** [comment]
Oui, il faudrait que l'on soit toutefois obligé de mettre à minima une information : OSM ou commune.
